clang/lib/Driver/ToolChain.cpp

```diff
@@ -1255,7 +1255,10 @@ llvm::opt::DerivedArgList *ToolChain::TranslateOffloadTargetArgs(
         continue;
       }
     }
-    XOffloadTargetArg->setBaseArg(A);
+
+    if (!XOffloadTargetArg)
+        XOffloadTargetArg->setBaseArg(A);
+
```

Comment:
```suggestion
    if (!XOffloadTargetArg)
        continue;
```
if there is no `XOffloadTargetArg`, we don't want to set the base arg or release (nothing to release).